client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -31,6 +36,22 @@ interface Props {
 
 const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+  const { email, password } = demoUser;
+
+  const handleDemoUserLogin = () => {
+    login(email, password).then((data) => {
+      if (data.error) {
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        updateLoginContext(data.success);
+      } else {
+        console.error({ data });
+        updateSnackBarMessage('An unexpected error occurred. Please try again');
+      }
+    });
+  };
```

Comment:
We shouldn't be repeating ourselves here. Maybe we can refactor this into the API layer?